pages/learn/integration/pre-built-integrations/mysql-to-salesforce-integration.js

```diff
@@ -235,7 +235,7 @@ export default function Learn({ samples, content }) {
               <Container>
                 <Row>
                   <Col xs={12} lg={6} style={{ fontSize: "18px" }}>
-                    <p>Data about products, customers, and sales transactions is often scattered across various systems,
+                    <p>Data about products, customers, and sales transactions are often scattered across various systems,
```

Comment:
I think both "is" and "are" are correct here. We have to pick the more readable one.

Reply:
This is what chatgpt says:


The term "data" can be both singular and plural, depending on its usage.

Singular: "Data" is often treated as a singular noun when referring to it as a collective or uncountable entity, such as "Data is important for decision-making."

Plural: However, it can also be treated as a plural noun when referring to multiple pieces or sets of information, like "The data are in various formats."

In your original sentence, "Data about products, customers, and sales transactions are often scattered across various systems," "data" is used in the plural form because it refers to multiple pieces of information about products, customers, and sales transactions.